src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -156,13 +179,48 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
     window.location.href = venueLink;
   }, [redirectionUrl, venueId]);
 
+  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
+
+  const [showRadioPopover, setShowRadioPopover] = useState(false);
+
   if (!venueId || !venue) return null;
 
   // TODO: ideally this would find the top most parent of parents and use those details
   const navbarTitle = parentVenue?.name ?? venue.name;
 
   const profileImage = profile?.pictureUrl || DEFAULT_PROFILE_IMAGE;
 
+  const renderSoundCloudIframe = () => {
+    const radioStation = !!hasRadioStations && radioStations![0];
```

Comment:
The `!!` of this should probably be done where it is set. And could use our `isTruthy` semantic wrapper.

Reply:
I see a lot of `!` usages, can this actually never be `undefined`? Otherwise it may just be avoiding our typechecking that could lead to errors at runtime.